go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/nikhita/github-contrib
+
+go 1.11
```

Comment:
Can you move this directly to 1.13 since that's the latest version?

Reply:
Isn't the `go` directive here used to specify the minimum go version?

I know you mention minimum go version as >=1.7 but for those using go<1.11, they'll not have any use for go.mod since it's not a feature in those versions.

So I thought adding go1.11 would be a good idea since people with those versions can still use the project.